elastalert/alerts.py

```diff
@@ -296,20 +296,30 @@ class StompAlerter(Alerter):
     """ The stomp alerter publishes alerts via stomp to a broker. """
     required_options = frozenset(['stomp_hostname', 'stomp_hostport', 'stomp_login', 'stomp_password'])
 
-    def alert(self, matches):
+    def check_if_match_exists(mes,matchkey):
```

Comment:
This function is a duplicate of util.lookup_es_key.

Reply:
Right, I will update my code to use util.lookup_es_key. Thx for the hint.